roles/ngi_reports/defaults/main.yml

```diff
@@ -1,10 +1,9 @@
 ---
 ngi_reports_repo: https://github.com/NationalGenomicsInfrastructure/ngi_reports.git
 ngi_reports_dest: "{{ sw_path }}/ngi_reports"
-ngi_reports_version: 51ffe95e2b06e11f289b39db5551d931f34b9cdb
+ngi_reports_version: 6b8b8a691d697c021eff5776bbb6b110cf376bca
 ngi_reports_log: "/log/ngi_reports.log"
 ngi_reports_log_sthlm: "{{ ngi_pipeline_sthlm_path }}/{{ ngi_reports_log }}"
-ngi_reports_log_upps: "{{ ngi_pipeline_upps_path }}/{{ ngi_reports_log }}"
```

Comment:
Maybe we don't have to remove this? 
Regarding ngi_reports in general, I told you before we don't use them anymore, but it was not entirely true. According to our BIO team we still run piper for users in case they are continuing an old project. I think it's kind of rare, but might still happen. 
Does the new version still support `ngi_reports ign_aggregate_report` and `ngi_reports ign_sample_report`? 
If not, I think the most convenient solution is for us to just load an old environment when running TACA.

Reply:
The ign reports have been removed from the latest version of ngi_reports so using an older deployment would be safest.